### PR TITLE
E-Commerce Build

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
The schema for latest e-commerce survey.

[Trello](https://trello.com/c/XdX6Qhwb/2509-e-commerce-build)
[Slides](https://docs.google.com/presentation/d/1z-6VcZ-njxqwozbxaCvMIT3nT0dKMcvl_GYd0a4YXwQ/edit)
[Quick Launch](https://preprod-new-surveys-launch.eq.ons.digital/quick-launch?url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/eq-2509-e-commerce-build/data/en/ecommerce_0001.json)

To do:
- ~~External link on intro page when the link is ready.~~
- ~~Change `q_code` for radio options with playback (Slides show multiple q_codes, radio can only have one). Waiting to hear from Donna.~~

Notes:
- Changed 1 decimal to 2 decimal places.
- Removed answer description where it references `decimal places`

---

### How to review 
Ensure:
- Everything matches slides.
- Routing is correct (I also changed author routing to reduce the no. of routing/skips, ie where author has inverse logic)

### CODE REVIEW

File: data/en/ecommerce_0001.json [hunk 1]
```diff
@@ -0,0 +1,2341 @@
...
+                            "id": "question1365",
+                            "title": "What is the maximum contracted download speed of the fastest fixed internet connection of the business?",
+                            "description": "",
+                            "type": "General",
+                            "answers": [{
+                                "id": "answer1831",
+                                "mandatory": true,
+                                "type": "Radio",
+                                "label": "",
+                                "description": "Please tick one box only",
```
Comment (bitdivision) [inline data/en/ecommerce_0001.json:595]: Should be qcode: 'r1'

File: data/en/ecommerce_0001.json [hunk 2]
```diff
@@ -0,0 +1,2341 @@
...
+                            "type": "General",
+                            "definitions": [{
+                                "title": "Definition of ICT (Information and Communication Technology)",
+                                "content": [{
+                                    "description": "Technology that provides access to information through telecommunications. It is similar to Information Technology (IT), but focuses primarily on communication technologies. This includes the internet, wireless networks, mobile phones, and other communication media."
+                                }]
+                            }],
+                            "answers": [{
+                                "id": "answer1861",
+
```
Comment (bitdivision) [inline data/en/ecommerce_0001.json:1612]: Should be qcode: r2

**Review by bitdivision — CHANGES_REQUESTED** [review]

File: data/en/ecommerce_0001.json (hunk 2 above)
Comment (bitdivision) [inline data/en/ecommerce_0001.json]: Changed to 487

File: data/en/ecommerce_0001.json (hunk 1 above)
Comment (MebinAbraham) [reply data/en/ecommerce_0001.json]: Donna confirmed or we making it up for time being?

File: data/en/ecommerce_0001.json (hunk 2 above)
Comment (MebinAbraham) [reply data/en/ecommerce_0001.json]: Whoops. Good spot

Comment (bitdivision) [reply data/en/ecommerce_0001.json:1612]: @MebinAbraham 	These were agreed with Donna.

**Comment by bitdivision** [comment]
Seems weird that they have section introductions for everything apart from the first?

I can't see any obvious issues at the moment, but will have another quick scan through.

**Review by bitdivision — APPROVED** [review]

File: data/en/ecommerce_0001.json [hunk 3]
```diff
@@ -2132,8 +2132,8 @@
                                 "label": "",
                                 "description": "",
                                 "options": [{
-                                        "label": "They’re not used",
```
Comment (bitdivision) [inline data/en/ecommerce_0001.json]: Hmm, this will affect SDX. Is it only this one?

Comment (MebinAbraham) [reply data/en/ecommerce_0001.json]: This is how it was originally in the slides...

**Comment by MebinAbraham** [comment]
Regarding percentage, Donna will confirm, whether it should be 1 or 2 decimal places, after speaking with the business.

File: data/en/ecommerce_0001.json (hunk 3 above)
Comment (bitdivision) [reply data/en/ecommerce_0001.json]: Ah

**Review by bitdivision — APPROVED** [review]
Double checked everything

**Review by dwyeradam — DISMISSED** [review]

**Comment by MebinAbraham** [comment]
Added the new website page and fixed an issue with the totaliser.
https://github.com/ONSdigital/eq-survey-runner/issues/1902

**Comment by MebinAbraham** [comment]
Since the JS has no context of the number of decimal places defined by the schema. The totaliser will always default with decimal places.

**Review by dwyeradam — APPROVED** [review]

**Pull request merged**